mod-source-record-manager-server/src/main/java/org/folio/dao/JobExecutionFilter.java

```diff
@@ -31,6 +32,13 @@ public class JobExecutionFilter {
   private String userId;
   private Date completedAfter;
   private Date completedBefore;
+  private boolean isDeleted;
+
+
+  public JobExecutionFilter isDeleted(boolean isDeleted) {
+    this.isDeleted=isDeleted;
```

Comment:
missing some spaces here: this.isDeleted = isDeleted;

Reply:
Hi @SerhiiNosko - This code was not required, Hence removed. Thank you 🙂